doc/user/install-operator-sdk.md

```diff
@@ -0,0 +1,75 @@
+# Install the Operator SDK CLI
+
+## Install from release
+
+### Linux
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
+```
+
+### OSX
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-apple-darwin && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
+```
+
+#### Verifying the downloaded binary
```

Comment:
Since Homebrew now claims support for Linux, I agree with making it a separate section outside of the macOS section. But it's a little awkward fitting the verification section in alongside, I think.

What do you think of the following outline?

## Install from GitHub release
### Download the release binary
```sh
# Linux
curl -Lo operator-sdk ...
# macOS
curl -Lo operator-sdk ...
```
### Verify the release binary
```sh
# Linux
curl -Lo operator-sdk.asc ...
# macOS
curl -Lo operator-sdk.asc ...

# Verify the binary
gpg --verify operator-sdk.asc
rm operator-sdk.asc
```
...

### Install the release binary in your PATH
``` 
chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
```
## Install from Homebrew
...

## Compile and install from master
...